File: lib-extension/src/main/java/com/xiaoyv/flexiflix/extension/model/FlexMediaPlaylist.kt
package com.xiaoyv.flexiflix.extension.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * [FlexMediaPlaylist]
 *
 * @author why
 * @since 5/8/24
 */
@Parcelize
data class FlexMediaPlaylist(
    var title: String = "默认",
    var items: List<FlexMediaPlaylistUrl> = emptyList()
) : Parcelable

File: web-nodejs/extension-common/src/model/FlexMediaPlaylist.ts
import {FlexMediaPlaylistUrl} from "./FlexMediaPlaylistUrl";

/**
 * 播放列表
 *
 * @author why
 * @since 5/8/24
 */
export interface FlexMediaPlaylist {
    /**
     * 播放列表标题
     */
    title: string

    /**
     * 播放列表的播放媒体条目
     */
    items: FlexMediaPlaylistUrl[]
}
